lib/data/wrapper.js

```diff
@@ -119,11 +119,6 @@ const data = {
         // If objectGetInfo.key exists the md-model-version is 2 or greater.
         // Otherwise, the objectGetInfo is just the key string.
         const key = objectGetInfo.key ? objectGetInfo.key : objectGetInfo;
-        log.debug('sending delete to datastore', {
```

Comment:
Let's keep this.  So we will have one log of all the keys intended to be deleted and then this with each key as it goes.